components/Footer.js

```diff
@@ -7,80 +7,124 @@ class Footer extends Component {
+            <Link href="/about-the-data">About the Data</Link>
+          </div>
+          <div className="section section__data-links">
+            <h4 class="section-title">Data</h4>
+            <Link href="/data">Explore the Data</Link><br></br>
+            <Link href="/publications">Publications</Link>
+          </div>
+          <div className="section section__get-involved-links">
+            <h4 class="section-title">Get Involved</h4>
+            <Link href="/contact">Volunteer</Link><br></br>
+            <Link href="/donate">Donate</Link><br></br>
+            <Link href="/contact">Contact Us</Link>
+          </div>
+          <div className="section section__social-links">
+            <h4 class="section-title">Follow Us</h4>
+            <a
+              href="https://www.facebook.com/TXJusticeInitiative"
+              target="_blank"
+              title="Visit Texas Justice Initiative on Facebook"
+            >
+              <img src={require('../images/tji-fb-icon-yellow.svg')} alt="TJI Facebook" />
+            </a>
+            <a href="https://twitter.com/JusticeTexas" target="_blank" title="Visit Texas Justice Initiative on Twitter">
+              <img src={require('../images/tji-twitter-icon-yellow.svg')} alt="TJI Facebook" />
+            </a>
+            <a href="https://github.com/texas-justice-initiative" target="_blank" title="Visit Texas Justice Initiative on Github">
+              <img src={require('../images/tji-github-logo-yellow.svg')} alt="TJI Github" />
+            </a>
+          </div>
         </div>
+        <hr></hr>
         <div className="site-info">
           <p>
-            Copyright 2018 Texas Justice Initiative. All rights reserved.{' '}
-            <a href="http://texasjusticeinitiative.org/disclaimer">Disclaimer</a>.
-          </p>
-          <p>
-            TJI <a href="http://texasjusticeinitiative.org/thanks/">appreciates</a> your creativity and talent.
+            TJI <a href="http://texasjusticeinitiative.org/thanks">appreciates</a> your creativity and talent. <Link href="/about">Learn more</Link> about the individuals and organizations who have helped us in our mission.
           </p>
         </div>
+        
       </StyledFooter>
     );
   }
 }
 
 const StyledFooter = styled.footer`
-  background-color: #ebebeb;
-  color: #0b5d93;
+  background-color: ${props => props.theme.colors.primaryBlue};
+  color: white;
   font-size: 12px;
   border-top: 1px solid #0b5d93;
   padding: 2em;
-  .newsletter-container {
-    font-family: Museo, Arial, Helvetica, sans-serif;
-    padding: 20px 50px;
-    .newsletter-cta {
-      font-size: 26px;
-    }
+  hr {
+    border-width: 0.4px;
+    width: 80%;
+    margin: 20px auto;
   }
-  #logo {
-    text-align: center;
-    img {
-      height: 100px;
+  .section-container {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-between;
+    padding: 2em 0;
+    max-width: 1028px;
```

Comment:
There are a few styles in here which we can use theme props for instead of hard coding. I think this and some of the colors used below have theme props already.

On that note, I think after launch another small, and tedious, project we can set out for someone to do is to update the spacing across the site with theme props.